sdk/src/main/java/com/hedera/hashgraph/sdk/MirrorNodeService.java

```diff
@@ -0,0 +1,199 @@
+/*-
+ *
+ * Hedera Java SDK
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.hedera.hashgraph.sdk;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.hedera.hashgraph.sdk.proto.TokenBalance;
+import com.hedera.hashgraph.sdk.proto.TokenFreezeStatus;
+import com.hedera.hashgraph.sdk.proto.TokenKycStatus;
+import com.hedera.hashgraph.sdk.proto.TokenRelationship;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+class MirrorNodeService {
+
+    private final MirrorNodeGateway mirrorNodeGateway;
+
+    MirrorNodeService(MirrorNodeGateway mirrorNodeGateway) {
+        this.mirrorNodeGateway = mirrorNodeGateway;
+    }
+
+    /**
+     * Get AccountId num from mirror node using evm address.
+     *
+     * @param evmAddress
+     * @return
```

Comment:
minor suggestion: the last line of these doc comments can be updated to say what the method actually returns
```suggestion
     * @return account id
```